packages/data-context/src/actions/CohortsActions.ts

```diff
@@ -1,7 +1,11 @@
 import type { Cohort } from '@packages/types'
 import type { DataContext } from '..'
+import { WEIGHTED, WEIGHTED_EVEN } from '../util/weightedChoice'
+const debug = require('debug')('cypress:data-context:cohorts')
 
 export interface CohortsApiShape {
+  getCohorts(): Promise<Record<string, Cohort>>
+
   getCohort(name: string): Promise<Cohort>
```

Comment:
I think there's a hole in the typings - couldn't this be nil if you try to get a cohort that hasn't been inserted yet? Not sure whether it makes sense to tighten up the typings in `server/lib/cohorts.ts` (everything coming out of there is defined as `any`), or better to just ensure the `CohortsApiShape` is properly reflecting the expected types

Reply:
Lachlan explained that part of the history of the code base was that the server was not well typed and was hard to introduce types, so part of the what the data-context package is doing is putting a typed layer on top of the server.  I will update the `CohortsApiShape` types to reflect what is expected.